go/services/statestore/options.go

```diff
@@ -35,6 +35,9 @@ type (
 	// WithManualAck allows notifications to be manually acknowledged.
 	WithManualAck bool
 
+	// WithPersist indicates that the key should be stored to disk.
+	WithPersist bool
```

Comment:
I'm unfamiliar with Go, but is the default value here false?

Reply:
Correct.